sortinghat/core/schema.py

```diff
@@ -77,6 +78,45 @@ def convert_json_field_to_generic_scalar(field, registry=None):
     return OperationArgsType(description=field.help_text, required=not field.null)
 
 
+def read_date_filter(filter_value):
+    """Extract the filter terms from a date filter
+
+    The accepted formats are controlled by regular expressions
+    matching two patterns: a comparison operator (>, >=, <, <=) and a date
+    OR a range operator (..) between two dates.
+
+    The accepted date format is ISO 8601, YYYY-MM-DDTHH:MM:SS
+
+    :param filter_value: String containing the filter value
+
+    :returns: A dictionary including an operator and the date values
+    """
+    # Accepted date format is ISO 8601, YYYY-MM-DDTHH:MM:SS
+    filter_data = {
+        "operator": None,
+        "date1": None,
+        "date2": None
+    }
+
+    # Filter with a comparison operator (>, >=, <, <=) and a date (e.g. `>=YYYY-MM-DDTHH:MM:SS`)
+    oper_comparison = r"^(<=?|>=?)(\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2})$"
```

Comment:
This doesn't allow to look for an exact date. What would be the way to implement it?

Reply:
As the accepted format I chose included date+time, it didn't make sense to me adding the `==` operator. I think the way it is done now, it could be set as a range filter like `YYYY-MM-DDT00:00:00..YYYY-MM-DDT23:59:59`, WDYT?